x/incentives/README.md

```diff
@@ -1,84 +0,0 @@
-# Incentives Module
```

Comment:
Assuming that this is to be replaced and updated via `spec/README.md`

Let's make sure we revisit some of these contents as these were for easier integration for external incentivizers.

Reply:
The specific gauge examples still lives in the spec folder under the "create gauge" tx example

https://github.com/osmosis-labs/osmosis/tree/main/x/incentives/spec#create-gauge-1

Do you think we should link this maybe in the intro so integrators can get to it easier?